s1_needle_pose_translation : [0.0, -0.00673698, 0.447744]

s1_needle_pose_orientation : [0.0, 0.0, 1.0, 0.35355]

s1_arm_index : 1

s1_grasp_pose_index : 3

s1_joint_values : [0.664327, 0.0919183, 0.069904, 0.00623299, 0.953258, -0.225286]

cs_needle_pose_translation : [0.0, -0.00673698, 0.447744]

cs_needle_pose_orientation : [0.0, 0.0, 1.0, 0.35355]

cs_arm_index : 2

cs_grasp_pose_index : 2

cs_joint_values : [0.664327, 0.0919183, 0.069904, 0.00623299, 0.953258, -0.225286]
## =====================================================
#psm_two_gripper:
#    end_effector_name: 'psm_two_gripper'  #ee group name
#
#    #actuated joints in end effector
#    joints : ['PSM2_jaw']
#
#    #open position
#    pregrasp_posture : [0.523]  #define gripper jaw's open angle
#    pregrasp_time_from_start : 4.0
#
#    #close position
#    grasp_posture : [0.262]  #define gripper jaw's close angle when hold object
#    grasp_time_from_start : 4.0
#
#    end_effector_parent_link: 'PSM2_tool_wrist_sca_shaft_link'
#
#    end_effector_tool_tip_link: 'PSM2_tool_tip_link'
#
#    #normal needle grasping parameters
#    theta_normal : [0.004, 0.0, 0.0, 0.0833]
#
#    #grasping parameters limits
#    theta_limits : [0.0, 0.005, -1.5708, 1.5708, 0.0, 6.283, 0.0, 3.1415]
#
#    #possible needle grasping search resolution
#    theta_resolution : [0.001, 0.628 ,1.047, 0.628]
#    #theta_resolution : [0.001, 0.2094, 0.2094, 0.2094]
#    #theta_resolution : [0.001, 0.2094, 0.1047, 0.2094]
#
#    approach_retreat_desired_dist : 0.008
#
#    approach_retreat_min_dist : 0.005
#
#    angle_resolution : 16
# '{transforms: {header: {frame_id: 'world'}, child_frame_id: 'needle', transform: {translation: {x: 0.0, y: -0.00673698, z: 0.447744}, rotation: {x: 0.0, y: 0.0, z: 0.0, w: 1.0}}}}'
